doc/data/messages/e/else-if-used/bad.py

```diff
@@ -0,0 +1,7 @@
+if input():
+    pass
+else:
+    if len(input()) >= 10:  # [else-if-used]
+        pass
+    else:
+        pass
```

Comment:
You're going to need a rc file to be able to load the extension here.

Reply:
Apologies for my ignorance, how should I go about using an .rc file in the workflow?